nx/public/plugins/exp/exp.js

```diff
@@ -56,6 +69,12 @@ function handleLoad({ target }) {
     if (e.data.ready) init(port1);
     if (e.data.reload) reloadPage();
     if (e.data.preview) previewExp(e.data);
+    if (e.data.reset) {
+      port1.close(); port2.close();
+      pollConnection(() => {
+        handleLoad({ target });
+      });
+    }
```

Comment:
We don't need this any more, right?

Reply:
We do. This is to build up an initial connection to the iframe as soon as it's ready. When the iframe reloads, we need to build a new connection. If you remove this, `window.location.reload` inside the iframe breaks it, as it gets stuck waiting for a connection.